Casks/propresenter-beta.rb

```diff
@@ -0,0 +1,31 @@
+cask "propresenter-beta" do
+  version "7.4.1,117702913"
+  sha256 "13c9bc59786d92dc085c7ee23c217745673f0df7861e7bdb420f137fa667be7b"
+
+  url "https://renewedvision.com/downloads/propresenter/mac/ProPresenter_#{version.before_comma}_#{version.after_comma}.zip"
+  appcast "https://api.renewedvision.com/v1/pro/upgrade?platform=macos&osVersion=0&appVersion=0&buildNumber=0&includeNotes=0&channel=beta"
+  name "ProPresenter"
+  desc "Presentation and production application for live events"
+  homepage "https://www.renewedvision.com/propresenter.php"
+
+  conflicts_with cask: "homebrew/cask/propresenter"
```

Comment:
```suggestion
  conflicts_with cask: "propresenter"
```

Since this is in the main tap.